src/api/cage.rs

```diff
@@ -253,6 +253,7 @@ pub struct CreateCageDeploymentIntentRequest {
     not_after: String,
     metadata: VersionMetadata,
     healthcheck: Option<String>,
+    desired_replicas: i32,
```

Comment:
Don't think this needs to be signed?
```suggestion
    desired_replicas: u32,
```